11-Sep-2023 18:06:54 - ERROR - DB_actions.py - 95 - Eroare la introducerea in datelor in baza de date. Query-ul a fost scris gresit. 
 syntax error at or near ")"
LINE 2: ...                                  values ('', '', '', '', );
                                                                     ^

11-Sep-2023 20:04:56 - INFO - DB_actions.py - 144 - User  a accesat baza de date
                pentru a modifica stockul din depozit pentru 0102C-13256, acesta fiind acum 1.
11-Sep-2023 20:05:04 - INFO - DB_actions.py - 144 - User  a accesat baza de date
                pentru a modifica stockul din depozit pentru 0102C-13256, acesta fiind acum 1.
11-Sep-2023 20:05:12 - INFO - DB_actions.py - 144 - User  a accesat baza de date
                pentru a modifica stockul din depozit pentru 0102C-13256, acesta fiind acum 2.
11-Sep-2023 20:08:34 - ERROR - DB_actions.py - 161 - Eroare la introducerea in datelor in baza de date. Query-ul a fost scris gresit. 
 unterminated quoted identifier at or near "" = '0102C-14003';"
LINE 3: ...                                WHERE "Part_number" = '0102C...
                                                             ^

11-Sep-2023 20:10:13 - INFO - DB_actions.py - 153 - User  a accesat baza de date
                pentru a modifica stockul din depozit pentru 0102C-14003, acesta fiind acum (9,).
11-Sep-2023 20:10:40 - INFO - DB_actions.py - 153 - User  a accesat baza de date
                pentru a modifica stockul din depozit pentru 0102C-14003, acesta fiind acum 10.
11-Sep-2023 20:12:58 - ERROR - DB_actions.py - 95 - Eroare la introducerea in datelor in baza de date. Query-ul a fost scris gresit. 
 syntax error at or near ")"
LINE 2: ...                       values ('0102C-14003', '', '', '', );
                                                                     ^

13-Sep-2023 20:03:01 - ERROR - DB_actions.py - 56 - Eroare la introducerea datelor in baza de date: invalid input syntax for type integer: ""
LINE 3:                             "KW" = '',
                                           ^

16-Sep-2023 12:59:23 - INFO - DB_actions.py - 46 - User  a accesat baza de date pentru a modifica datele masinii: B24AEN, WF04XXWPD46R72874.
16-Sep-2023 13:00:41 - INFO - DB_actions.py - 46 - User  a accesat baza de date pentru a modifica datele masinii: B51TSV, WNUSFJ12U3N445556.
16-Sep-2023 14:11:30 - INFO - login_sequence.py - 50 - Utilizatorul necula77 s-a logat cu succes.
16-Sep-2023 14:11:51 - INFO - login_sequence.py - 50 - Utilizatorul necula77 s-a logat cu succes.
16-Sep-2023 14:12:02 - INFO - login_sequence.py - 50 - Utilizatorul necula77 s-a logat cu succes.
16-Sep-2023 14:12:14 - INFO - login_sequence.py - 50 - Utilizatorul necula77 s-a logat cu succes.
16-Sep-2023 14:12:23 - INFO - login_sequence.py - 50 - Utilizatorul necula77 s-a logat cu succes.
16-Sep-2023 14:12:34 - INFO - login_sequence.py - 50 - Utilizatorul necula77 s-a logat cu succes.
16-Sep-2023 14:13:15 - INFO - login_sequence.py - 50 - Utilizatorul sefu s-a logat cu succes.
16-Sep-2023 18:38:23 - ERROR - login_sequence.py - 85 - Eroare la inregistrare: column "authorization_level" of relation "LOGIN_INFO" does not exist
LINE 2:                                 "function", authorization_le...
                                                    ^

16-Sep-2023 18:39:13 - ERROR - login_sequence.py - 85 - Eroare la inregistrare: function gen_salt(unknown) does not exist
LINE 3: ...         VALUES('Mircea', 'Cioaca', crypt('sefu', gen_salt('...
                                                             ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.

16-Sep-2023 18:39:52 - ERROR - login_sequence.py - 85 - Eroare la inregistrare: syntax error at or near ";"
LINE 4:                                  'gsdv', CURRENT_DATE);
                                                              ^

16-Sep-2023 18:40:27 - ERROR - login_sequence.py - 85 - Eroare la inregistrare: syntax error at or near ";"
LINE 4:                                  'fdscf', CURRENT_DATE;
                                                              ^

16-Sep-2023 18:40:46 - ERROR - login_sequence.py - 85 - Eroare la inregistrare: new row for relation "LOGIN_INFO" violates check constraint "LOGIN_INFO_function_check"
DETAIL:  Failing row contains (3, Mircea, sadfm, sefu, 123, dmskf, 2023-09-16).

16-Sep-2023 18:41:39 - INFO - login_sequence.py - 76 - User: sefu a fost adaugat in baza de date, NUME: fas PRENUME: Mircea.
16-Sep-2023 18:42:08 - INFO - login_sequence.py - 53 - Utilizatorul sefu s-a logat cu succes.
16-Sep-2023 18:43:42 - ERROR - login_sequence.py - 85 - Eroare la inregistrare: new row for relation "LOGIN_INFO" violates check constraint "LOGIN_INFO_function_check"
DETAIL:  Failing row contains (5, , , , , , 2023-09-16).

16-Sep-2023 18:44:29 - ERROR - login_sequence.py - 85 - Eroare la inregistrare: new row for relation "LOGIN_INFO" violates check constraint "LOGIN_INFO_function_check"
DETAIL:  Failing row contains (6, , , , , , 2023-09-16).

16-Sep-2023 18:44:44 - INFO - login_sequence.py - 76 - User: diosf a fost adaugat in baza de date, NUME: dskmfo PRENUME: asmkd.
16-Sep-2023 18:44:45 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:46 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:47 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:47 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:47 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:48 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:48 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:48 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:48 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:48 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:49 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:44:49 - ERROR - login_sequence.py - 80 - Username-ul: diosf, exista deja, va rugam sa alegeti altul.
16-Sep-2023 18:45:12 - INFO - login_sequence.py - 76 - User: noknfdsa a fost adaugat in baza de date, NUME: kmidofs PRENUME: afskma.
16-Sep-2023 19:52:05 - ERROR - login_sequence.py - 85 - Eroare la inregistrare: new row for relation "LOGIN_INFO" violates check constraint "LOGIN_INFO_function_check"
DETAIL:  Failing row contains (21, , , , , , 2023-09-16).

16-Sep-2023 19:55:38 - ERROR - login_sequence.py - 84 - s
16-Sep-2023 19:55:46 - ERROR - login_sequence.py - 84 - s
16-Sep-2023 19:56:04 - ERROR - login_sequence.py - 84 - new row for relation "LOGIN_INFO" violates check constraint "LOGIN_INFO_function_check"
DETAIL:  Failing row contains (25, sad, asfcas, das, das, asc, 2023-09-16).

16-Sep-2023 19:56:15 - INFO - login_sequence.py - 76 - User:  a fost adaugat in baza de date, NUME:  PRENUME: .
16-Sep-2023 19:59:14 - ERROR - login_sequence.py - 84 - new row for relation "LOGIN_INFO" violates check constraint "LOGIN_INFO_function_check"
DETAIL:  Failing row contains (27, , , , , , 2023-09-16).

16-Sep-2023 20:01:13 - ERROR - login_sequence.py - 84 - new row for relation "LOGIN_INFO" violates check constraint "LOGIN_INFO_function_check"
DETAIL:  Failing row contains (28, , , , , , 2023-09-16).

16-Sep-2023 20:01:41 - ERROR - login_sequence.py - 84 - Campurile nu au fost completate corect!
16-Sep-2023 20:03:47 - INFO - login_sequence.py - 76 - User: sdf a fost adaugat in baza de date, NUME: sdfc PRENUME: asdfsdfvadfgdsagvwaefsvdsafvsdafv.
16-Sep-2023 20:08:17 - INFO - login_sequence.py - 76 - User: asd a fost adaugat in baza de date, NUME: asd PRENUME: asfceafaewgfvafgvdsfvdsfdsfvs.
16-Sep-2023 20:08:58 - ERROR - login_sequence.py - 84 - Campurile nu au fost completate corect!
16-Sep-2023 20:09:01 - ERROR - login_sequence.py - 84 - Campurile nu au fost completate corect!
16-Sep-2023 20:09:21 - ERROR - login_sequence.py - 84 - Campurile nu au fost completate corect!
16-Sep-2023 20:09:22 - ERROR - login_sequence.py - 84 - Campurile nu au fost completate corect!
17-Sep-2023 18:15:16 - ERROR - login_sequence.py - 84 - Campurile nu au fost completate corect!
17-Sep-2023 18:15:23 - INFO - login_sequence.py - 76 - User: adc a fost adaugat in baza de date, NUME: awd PRENUME: sada.
17-Sep-2023 18:28:26 - INFO - login_sequence.py - 53 - Utilizatorul necula77 s-a logat cu succes.
17-Sep-2023 19:23:20 - INFO - login_sequence.py - 143 - User: 4 a fost editat in baza de date,
                         NUME: Mircea PRENUME: Andrei USERNAME: sefu.
17-Sep-2023 19:25:52 - INFO - login_sequence.py - 143 - User: 4 a fost editat in baza de date,
                         NUME: Andrei PRENUME: Mircea USERNAME: sefu.
17-Sep-2023 19:27:51 - INFO - login_sequence.py - 143 - User: 4 a fost editat in baza de date,
                         NUME: Mircea PRENUME: Andrei USERNAME: sefu.
17-Sep-2023 19:29:33 - INFO - login_sequence.py - 143 - User: 4 a fost editat in baza de date,
                         NUME: Andrei PRENUME: Mircea USERNAME: sefu.
17-Sep-2023 19:30:27 - INFO - login_sequence.py - 143 - User: 4 a fost editat in baza de date,
                         NUME: Andrei PRENUME: Mircea USERNAME: sefu.
17-Sep-2023 19:41:28 - ERROR - login_sequence.py - 149 - Username-ul: necula77, exista deja, va rugam sa alegeti altul.
17-Sep-2023 19:41:51 - INFO - login_sequence.py - 143 - User: 38 a fost editat in baza de date,
                         NUME: Mircea PRENUME: Andrei USERNAME: sefu.
17-Sep-2023 21:39:01 - INFO - login_sequence.py - 76 - User: sefu a fost adaugat in baza de date, NUME: Andrei PRENUME: Mircea.
17-Sep-2023 21:44:52 - INFO - login_sequence.py - 76 - User: sefu a fost adaugat in baza de date, NUME: Andrei PRENUME: Mircea.
17-Sep-2023 21:46:24 - INFO - login_sequence.py - 76 - User: sefu a fost adaugat in baza de date, NUME: Andrei PRENUME: Mircea.
17-Sep-2023 21:46:50 - INFO - login_sequence.py - 152 - User: 41 a fost editat in baza de date,
                         NUME: Mihaela PRENUME: Andreea USERNAME: sefu.
17-Sep-2023 21:47:18 - ERROR - login_sequence.py - 158 - Username-ul: necula77, exista deja, va rugam sa alegeti altul.
17-Sep-2023 21:47:21 - ERROR - login_sequence.py - 158 - Username-ul: necula77, exista deja, va rugam sa alegeti altul.
17-Sep-2023 21:47:43 - ERROR - login_sequence.py - 158 - Username-ul: necula77, exista deja, va rugam sa alegeti altul.
17-Sep-2023 21:48:05 - INFO - login_sequence.py - 154 - User: 41 a fost editat in baza de date,
                         NUME: Mihaela PRENUME: Andreea USERNAME: sefu.
17-Sep-2023 21:48:24 - ERROR - login_sequence.py - 160 - Username-ul: necula77, exista deja, va rugam sa alegeti altul.
17-Sep-2023 22:06:48 - INFO - login_sequence.py - 53 - Utilizatorul necula77 s-a logat cu succes.
